leetcode/array.md

```diff
@@ -0,0 +1,16 @@
+# Arrays
```

Comment:
Your pull request: LEETCODE-1022 Added transpose-matrix


Your commit: LEETCODE-1022 Added transpose-matrix

Your commit: LEETCODE-1022 Refactored transpose-matrix
Action of your comment must be from ['Added', 'Deleted']